examples/device/simple_spmv.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#include <spblas/spblas.hpp>
+
+#include <thrust/device_vector.h>
+
+#include <fmt/core.h>
+#include <fmt/ranges.h>
+
+int main(int argc, char** argv) {
+  using namespace spblas;
+
+  using T = float;
+  using I = spblas::index_t;
+
+  spblas::index_t m = 100;
+  spblas::index_t n = 100;
+  spblas::index_t nnz_in = 10;
+
+  fmt::print("\n\t###########################################################"
+             "######################");
+  fmt::print("\n\t### Running SpMV Example:");
+  fmt::print("\n\t###");
+  fmt::print("\n\t###   y = alpha * A * x");
+  fmt::print("\n\t###");
+  fmt::print("\n\t### with ");
+  fmt::print("\n\t### A, in CSR format, of size ({}, {}) with nnz = {}", m, n,
+             nnz_in);
+  fmt::print("\n\t### x, a dense vector, of size ({}, {})", n, 1);
+  fmt::print("\n\t### y, a dense vector, of size ({}, {})", m, 1);
+  fmt::print("\n\t### using float and spblas::index_t (size = {} bytes)",
+             sizeof(spblas::index_t));
+  fmt::print("\n\t###########################################################"
+             "######################");
+  fmt::print("\n");
+
+  auto&& [values, rowptr, colind, shape, nnz] =
+      generate_csr<T, spblas::index_t>(m, n, nnz_in);
+
+  thrust::device_vector<T> d_values(values);
+  thrust::device_vector<I> d_rowptr(rowptr);
```

Comment:
We need a different offset type name. In rocSPARSE, we usually use I for the offset and  J for the column indices.